components/basecomponents/MemoryGame/useTracking.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useRef } from 'react';
+import { useLanguage } from 'utils/useLanguageHook';
+import { usePlausible } from 'next-plausible';
+import { usePlatform } from 'utils/usePlatform';
+import { Platform } from '@types';
+import { useGameStore, Scene } from './gameStore';
+
+/* eslint-disable no-console */
+
+export const useTracking = () => {
+  const scene = useGameStore((state) => state.scene);
+  const currentThemeIndex = useGameStore((state) => state.currentThemeIndex);
+  const selectedCards = useGameStore((state) => state.selectedCards);
+
+  const plausible = usePlausible();
+  const platform = usePlatform();
+  const isKiosk = platform === Platform.KIOSK;
+  const { currentLanguage } = useLanguage();
+
+  const enabledAnalytics = useRef(true);
+
+  useEffect(() => {
+    const selected = enabledAnalytics.current && !!selectedCards.first;
```

Comment:
Tady bych asi zvolil jiný název pro enabledAnalytics, třeba enabledGameStartTracking. A dal bych to rovnou do podmínky .... if (enabledGameStartTracking && !!selectedCards.first)

Reply:
Děkuji, přejmenuji proměnnou.